launcher/CMakeLists.txt

```diff
@@ -654,6 +654,8 @@ SET(LAUNCHER_SOURCES
     ui/MainWindow.cpp
     ui/InstanceWindow.h
     ui/InstanceWindow.cpp
+    ui/WinDarkmode.h
+    ui/WinDarkmode.cpp
```

Comment:
It's probably best to have those two definitions inside a `#ifdef Q_OS_WIN` rather than having them spread in those two files, and `#ifdef` the include in Application.cpp as well imo